### PR TITLE
specifications for view

### PR DESCRIPTION
**Pull request by dcastroeyss**

### CODE REVIEW

File: card/mod/admin/set/self/admin_info.rb [hunk 1]
```diff
@@ -26,10 +26,10 @@ def warning_list warnings
   end
 
   def warning_list_with_auto_scope warnings
-      # 'ADMINISTRATOR WARNING'
-      admin_warn = tx(:admin_warn)
-      "<h5>#{admin_warn}</h5>" + warnings.join("\n")
-    end
+    # 'ADMINISTRATOR WARNING'
+    admin_warn = tx(:admin_warn)
```
Comment (ethn) [inline card/mod/admin/set/self/admin_info.rb]: is this supposed to be "tr"?

File: card/mod/admin/spec/set/self/admin_info_spec.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# -*- encoding : utf-8 -*-
+
+describe Card::Set::Self::AdminInfo do
+  # SPECSTUB
```
Comment (ethn) [inline card/mod/admin/spec/set/self/admin_info_spec.rb]: SPECSTUB means there aren't any specs yet.  So we can delete these.

File: card/mod/admin/set/self/admin_info.rb (hunk 1 above)
Comment (dcastroeyss) [reply card/mod/admin/set/self/admin_info.rb]: it looks like it is

File: card/mod/admin/spec/set/self/admin_info_spec.rb (hunk 2 above)
Comment (dcastroeyss) [reply card/mod/admin/spec/set/self/admin_info_spec.rb]: ok

File: card/mod/admin/spec/set/self/admin_info_spec.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# -*- encoding : utf-8 -*-
+
+describe Card::Set::Self::AdminInfo do
```
Comment (xithan) [inline card/mod/admin/spec/set/self/admin_info_spec.rb]: `describe` on the top level still works but it's a deprecated pattern that RSpec pollutes the global namespace like that. The better way is `RSpec.describe`.
IDE likes Rubymine also expect it that way.

fyi you can generate these files with `bundle exec decko generate card:set admin self admin_info --spec-only`

Comment (dcastroeyss) [reply card/mod/admin/spec/set/self/admin_info_spec.rb]: `RSpec.describe Card::Set::Self::AdminInfo do....`
Yes?

Comment (ethn) [reply card/mod/admin/spec/set/self/admin_info_spec.rb]: yes.

Comment (ethn) [reply card/mod/admin/spec/set/self/admin_info_spec.rb]: Yes.

File: card/mod/admin/spec/set/self/trash_spec.rb [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# -*- encoding : utf-8 -*-
+
+describe Card::Set::Self::Trash do
+  # SPECSTUB
```
Comment (ethn) [inline card/mod/admin/spec/set/self/trash_spec.rb]: (there are still a few of these)

**Pull request merged**